mmcv/runner/base_module.py

```diff
@@ -1,10 +1,27 @@
 # Copyright (c) Open-MMLab. All rights reserved.
 import warnings
 from abc import ABCMeta
+from collections import defaultdict
 
 import torch.nn as nn
 
-from mmcv import ConfigDict
+from mmcv.utils.logging import logger_initialized, print_log
+
+
+def update_init_infos(module, *, init_info):
```

Comment:
This method can be implemented as a private method of `BaseModule`, as it is highly coupled with the module.